convert-hf-to-gguf.py

```diff
@@ -323,12 +327,16 @@ def write_tensors(self):
 
     def write(self):
         self.write_tensors()
+        self.gguf_writer.init_shards()
```

Comment:
The type of `self.gguf_writer` is `GGUFWriter` which does not have the `init_shards` method.

Reply:
Right, will fix that type annotation. Do we then want to rename `self.gguf_writer` itself or can we leave it as such?

Reply:
Leaving it as `self.gguf_writer` should be fine.

Reply:
Thanks. Fixed in c6ae1d6.